src/specsynthase/specbuilder.py

```diff
@@ -5,10 +5,7 @@
 class SpecBuilder(OrderedDict):
     def __init__(self, *args, **kwargs):
         OrderedDict.__init__(self, *args, **kwargs)
-        self._paths_key = "paths"
-        self._definitions_key = "definitions"
-        self._parameters_key = "parameters"
-        self._response_key = "responses"
+        self._mergeable_keys = ["definitions", "parameters", "paths", "responses", "securityDefinitions"]
```

Comment:
I think about this too, but the overridable logic should change a little to do this, because the dictionary update function doesn't makes a composable result, and all keys that we override today are composable.

Eg.:

`dict1 = {'key1': {'key1.1': 1, 'key1.2': 2}}
dict2 = {'key1': {'key1.3': 3, 'key1.4': 4}}

dict1.update(dict2)

#Result
{'key1': {'key1.3': 3, 'key1.4': 4}}`